frontend/src/routes/Search/searchDefinitions.tsx

```diff
@@ -731,10 +892,18 @@ export function CreateGlobalSearchDetailsLink(props: { item: any }) {
         `?${encodeURIComponent('showClusterIssues=true')}`
       )
     }
+    case 'virtualmachine':
+    case 'virtualmachineinstance': {
+      return createGlobalVMDetailsLink(
+        item,
+        acmExtensions,
+        generateDefaultSearchLink,
+        resourceRoutesResolved,
+        getDirectResourceRouteHandler
+      )
+    }
```

Comment:
I don't think we should be touching `CreateGlobalSearchDetailsLink` at all. If I understand correctly, this would be a link to a search result on a different hub cluster, so we have no way of knowing if that hub cluster has CNV installed and special pages for virtual machines. I think we should just leave this case alone. @zlayne does that make sense?